### PR TITLE
feat: introduce diff module

### PR DESCRIPTION
**Pull request by podarok**
Original Issue, this PR is going to fix: https://yusaslackinstance.slack.com/archives/C03E6SQQ0Q5/p1693238759352319?thread_ts=1693235674.555889&cid=C03E6SQQ0Q5

Make sure these boxes are checked before asking for review of your pull request - thank you!

If there is a new feature or this is a bug fix - use 9.x-2.x branch. We'll tag for release if the bug is critical asap or tag for release next bug fix release until critical issue arrived.

## Steps for review

- [ ] Please provide steps for review here.
- [ ] Please provide steps for review here.

## General checks
- [ ] All coding styles are fulfilled and there are no any issues reported by CodeSniffer. See [Code of Conduct](https://github.com/ymcatwincities/openy/wiki/Open-Y-Code-of-Conduct-and-Best-Practices).
- [ ] Try to use Conventional commit messages accordingly to the standard https://www.conventionalcommits.org/en/v1.0.0/#specification
- [ ] [Documentation](https://github.com/ymcatwincities/openy/tree/9.x-2.x/docs) has been updated according to PR changes.
- [ ] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [ ] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ymcatwincities/openy/blob/9.x-2.x/docs/Development/Upgrade%20path.md).
- [ ] Make sure your git email is associated with account on drupal.org, otherwise you won't get commits there. <br/>drupal.org email
- [ ] If you would like to get credits on drupal.org, [check documentation](https://github.com/ymcatwincities/openy/blob/9.x-2.x/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

### CODE REVIEW

**Comment by froboy** [comment]
@podarok this should have been added with https://github.com/open-y-subprojects/openy_features/pull/67/files. I think I forgot the version bump.

File: composer.json [hunk 1]
```diff
@@ -34,6 +34,7 @@
         "drupal/custom_formatters": "^3.0 || ^3.0@beta || ^4 || ^4.0@beta",
         "drupal/datalayer": "^1.0 || ^2.0 || ^2.0@RC",
         "drupal/daxko": "^1.0",
+        "drupal/diff": "^1.1",
```
Comment (froboy) [inline composer.json]:
```suggestion
```

**Comment by podarok** [comment]
> @podarok this should have been added with https://github.com/open-y-subprojects/openy_features/pull/67/files. I think I forgot the version bump.

see above commit, true

**Review by froboy — APPROVED** [review]

**Comment by froboy** [comment]
It looks like http://openy.cibox.tools:8081/view/GITHUB/job/INSTALL_PROFILE/2143/console is failing due to possibly an order of operations issue. I'm assuming `openy_features` might get installed too late? Checking

**Comment by podarok** [comment]
> It looks like http://openy.cibox.tools:8081/view/GITHUB/job/INSTALL_PROFILE/2143/console is failing due to possibly an order of operations issue. I'm assuming `openy_features` might get installed too late? Checking

You need to make some module depends from diff or add diff to openy.info.yml as default depedency ( no ), or better add it to some package we do install

**Comment by froboy** [comment]
@podarok ok, the above PR should resolve it. once it's merged and tagged I'll retest. Thanks.

**Pull request merged**